### DISCUSSION TITLE
Coolwallet pro dev card not generating signature with Polkadot sample script + argument

### DISCUSSION THREAD

**Discussion by samleong-wits** [CoolBitX-Technology/coolwallet-signing-tools#102] [Q&A]
Hello I have a dev card and I would like to make sure it is working correct. However, I am unable to generate signature based on the sample script and argument given in the README. 
E.g. 
Per [Polkadot](https://github.com/CoolBitX-Technology/coolwallet-signing-tools/tree/main/script-tester#polkadot)
Script: `03020E01C7070000000162CAA0C70002CC071000CAAC570002A2ACD70022FFF6CAACD7002CFFFBA2ACD70031FFFBA2ACD70036FFFBCAACC7003B04CAACC7003F04CAAC570043CAAC570063DC07C003444f54CC0FC00753533538505245CC0F1000CAAC5F00025AF09FC00FBAFCCE6C07230804DDE09700DAACD7C022FFF60AD207CC05065052455353425554546f4e`
Argument: 
`15328000002C8000016280000000000000000000000005008eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a480000000000BEEBCF08000000009999000000000000000000009a0200004d01000091b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c35c697847b25d385178aa150d29e5ce212339c5624183f74bdf45f4912c89749a`

When i clicked the GET, I expect to receive a signature. However, I got the following error: 
![image](https://user-images.githubusercontent.com/104709835/166260957-afc0c1dc-c6fe-45a5-a4f3-b2ca043bd65a.png)

![image](https://user-images.githubusercontent.com/104709835/166262719-28968730-4ba3-4b2d-84c3-db65cd388a7b.png)

I observed the following
1. I updated the firmware via android app, and I assumed I do not have to update with the script tester app. Nevertheless just in case, I clicked the UPDATE in the script tester app, I and saw a popup window with "Error!", no log being observed. What is the token value that I should use? 

2. I clicked register in the script tester app - but card shows paired error. I am getting the same error as seen in the above image when i tried to generate a signature with sample script. 

What are the steps that I might be missing?

**Comment by hank121314** [comment]
All these issues are caused by upgrade failure, we have provided a token for upgrading, please enter the token and upgrade again! Thanks!
